static/js/panels/ProfileForm.tsx

```diff
@@ -28,7 +43,49 @@ const ProfileForm: FC = () => {
     },
     validationSchema: ProfileSchema,
     onSubmit: (values) => {
-      createProfile(values.name, values.description)
+      // create LxdDevices object
+      const devicesObj: LxdDevices = {};
+      if (nicDevice?.network) {
+        devicesObj[nicDevice.name === "" ? nicDevice.network : nicDevice.name] =
```

Comment:
Why fall back to the network name as device name? This is a little wired, because the front-end doesn't explain this behaviour.

Reply:
Seemed like a reasonable default behaviour, considering that it is going to be applied only if the user has selected a network from the list. Of course any alternative proposal is always welcome. As for the lack of explanation... Well, also when you create an instance and leave the name field blank you find yourself with an instance named `freaky-panda` with no explanation anywhere... ;)

Reply:
True, let's add explanatory text in both places. I think the help prop on the Input component is a good place for this. Though it is only useful to the user once. Maybe let's discuss it with UX.

Reply:
I was thinking that maybe it could go in the `placeholder` as well, together with the example (e.g. eth0), so that it doesn't take further space. What do you think?